### PR TITLE
Added paypal and gitter badges

### PR DESCRIPTION
**Pull request by LarryBattle**
Added paypal and gitter badges because that's what the other repos have. :-)

https://github.com/TheAlgorithms/Python/blob/master/README.md
https://github.com/TheAlgorithms/Rust/blob/master/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # The Algorithms - Go
 
+[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/TheAlgorithms/100) &nbsp;
```
Comment (Panquesito7) [inline README.md:3]: We use [**Liberapay**](https://liberapay.com/TheAlgorithms) now. You may want to change it. 🙂
```suggestion
[![Income](https://img.shields.io/liberapay/receives/TheAlgorithms.svg?logo=liberapay)](https://liberapay.com/TheAlgorithms) &nbsp;
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/TheAlgorithms) &nbsp;
```
Comment (Panquesito7) [inline README.md:4]: Gitter is not really being used, so you may want to add a Discord badge, too.
```suggestion
[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/TheAlgorithms) &nbsp;
[![Discord chat](https://img.shields.io/discord/808045925556682782.svg?logo=discord&colorB=7289DA)](https://discord.gg/c7MnfGFGa6)
```

**Comment by raklaptudirm** [comment]
Resolve the problems and re-request review please.

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
Please make the changes requested by @Panquesito7. There have been some changes in the README.md file in the master branch. Could you pull our master branch code and resolve any conflicts that arise.

**Comment by LarryBattle** [comment]
Hey guys, sorry but I can't contribute to open source projects without getting approval from my employer :-( Closing for now.

**Comment by raklaptudirm** [comment]
What, how is that a thing?

**Pull request closed without merging**